.github/workflows/shared-vulnerability-scan-failure-notify.yaml

```diff
@@ -41,6 +48,8 @@ jobs:
     steps:
       - name: Checkout repo
         uses: actions/checkout@v4
+        with:
```

Comment:
This is needed due to https://github.com/IABTechLab/uid2-client-net/actions/runs/14483117083